service/src/main/java/uk/gov/hmcts/reform/fpl/utils/CaseConverter.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.fpl.utils;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+
+import java.util.Map;
+
+@Component
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class CaseConverter {
```

Comment:
Not sure wether this should be a component or just added to the abstract controller class

Reply:
It feels like a lot of extra code which could simply be done in the abstract class

Reply:
agree with you on this.